docs/index.asciidoc

```diff
@@ -430,6 +430,8 @@ Url to use when connecting to the JMS provider. This is only relevant for non-JN
 
 Name of the destination queue or topic to use.
 
+* Note that in some cases, it might be necessary to specify the `destination` in the format `<JMS module name>!<JMS queue name>`
```

Comment:
```suggestion
TIP: If the destination setting doesn't appear to be working properly, try this format: `<JMS module name>!<JMS queue name>`.
```

Reply:
If we're offering alternative formatting suggestions, perhaps we should be clear on what the usual format is.